packages/pnpm/test/cli.ts

```diff
@@ -1,4 +1,5 @@
 import prepare from '@pnpm/prepare'
+import { promises as fs } from 'fs'
```

Comment:
`createReadStream` is not available on the `node:fs/promises` module:
```suggestion
import { createReadStream, promises as fs } from 'fs'
```